src/server/stock_market/stock_market_controller.py

```diff
@@ -0,0 +1,19 @@
+from flask import Blueprint
+
+
+class StockMarketController:
+    """Stock Market Controller class."""
+
+    blueprint = Blueprint("stock", __name__, url_prefix="/stock")
```

Comment:
Same as with `AuthController`, can this be changed to private and then added to `@property` method with read-only access?